packages/core/src/WorkflowDownloadButton.tsx

```diff
@@ -22,6 +23,6 @@ export const WorkflowDownloadButton = (): JSX.Element => {
   }
 
   return (
-    <button className='btn btn-primary' onClick={downloadWorkflow}>Download archive</button>
+    <button className='btn btn-outline-primary' onClick={downloadWorkflow} disabled={hasErrors}>Download</button>
```

Comment:
I think there is no need to enable download when form is invalid. This will result in storing invalid values as these are in the data. I think it is better to leave it as it is now but if majority of users think that is better to allow download we can do it.